packages/chart/src/components/LinearChart.jsx

```diff
@@ -428,7 +430,27 @@ const LinearChart = props => {
                             </Text>
                           )}
 
-                          {orientation === 'vertical' && visualizationType !== 'Paired Bar' && !config.yAxis.hideLabel && (
+                          {orientation === 'vertical' && visualizationType === 'Bump Chart' && !config.yAxis.hideLabel && (
```

Comment:
Review the diff of the logic here. Instead of changing the previous logic you might consider adding an or statement to work with the Bump Chart.